### ISSUE TITLE
关于训练的损失函数

### ISSUE THREAD

**Issue by lijierhhh** [ZhangDY827/MHAN#2]
你好，最近在学习超分辨的内容，想试着跑通您论文中的代码      
```
         out1,out2 = model(input)
        # print(prediction[0])
        # print(prediction.size)
        # print(target.size)  
        #loss = criterion(prediction, target)
        loss1 = criterion(out1, target)
        loss2 = criterion(out2, target)
        loss = 10 * loss1 + loss2
```
这段代码中，提醒模型只有一个输出
`        out1,out2 = model(input)`
是否是输出同一个结果计算损失函数
```
        out1 = model(input)
        loss1 = criterion(out1, target)
        loss2 = criterion(out1, target)
        loss = 10 * loss1 + loss2
```
这样改是对的吗？

**Comment by ZhangDY827** [comment]
如果只是训练论文里面的方法的话
prediction = model(input)
\# print(prediction.size)
\# print(target.size)  
loss = criterion(prediction, target)
这就够了，应该是其他什么模型有两个输出的才这么做。我后面忘记改回来了。

**Comment by lijierhhh** [comment]
好的十分感谢您的解答
